include/rdma/fabric.h

```diff
@@ -392,6 +416,7 @@ struct fi_domain_attr {
 	size_t 			auth_key_size;
 	size_t			max_err_data;
 	size_t			mr_cnt;
+	struct fi_traffic_class traffic_class;
```

Comment:
The man pages should indicate that all new fields are only available starting in release 1.9.  This change updates the ABI, so there should be corresponding updates to abi_1_0.c

Reply:
Agreed.

Reply:
I've added the ABI changes. I'll be working on the man page changes soon.